saleor/static/dashboard-next/auth/components/SectionRoute.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from "react";
+import { Route, RouteProps } from "react-router-dom";
+
+import { UserContext } from "..";
+import AppRoot from "../../AppRoot";
+import NotFound from "../../NotFound";
+
+interface SectionRouteProps extends RouteProps {
+  resource?: string;
+}
+
+export const SectionRoute: React.StatelessComponent<SectionRouteProps> = ({
```

Comment:
Section as dashboard's section - like products, orders etc. Since we got only `manage_section` permissions, we can block whole section instead of atomic views.